inc/render-partials.php

```diff
@@ -0,0 +1,192 @@
+	$logo_template = '<img src="%1$s" alt="%2$s" />';
+
+	// First, check for Preset Logo Selection.
+	if ( $logo_select && 'none' !== $logo_select ) {
+		// load array of endorsed units.
+		$endorsed_logos = uds_wp_theme_get_endorsed_unit_logos();
+
+		// lookup logo filename.
+		$filename = '';
+		foreach ( $endorsed_logos as $unit ) {
+			if ( $unit['slug'] === $logo_select ) {
+				$filename = $unit['filename'];
+				break;
+			}
+		}
+		echo wp_kses( sprintf( $logo_template, get_template_directory_uri() . '/img/endorsed-logo/' . $filename, get_bloginfo( 'name' ) . ' Logo', home_url( '/' ) ), wp_kses_allowed_html( 'post' ) );
+
+		// Else, check for Logo URL.
+	} elseif ( $logo_url && '' !== $logo_url ) {
+		echo wp_kses( sprintf( $logo_template, $logo_url, get_bloginfo( 'name' ) . ' Logo', home_url( '/' ) ), wp_kses_allowed_html( 'post' ) );
+	}
+}
+
+/**
+ * Render the 'Contribute' button
+ *
+ * Takes the contribute_url setting and coditionally renders the button.
+ */
+function uds_wp_render_contribute_button() {
+	$contribute_url = get_theme_mod( 'contribute_url' );
+	$contribute_template = '<p class="contribute-button"><a href="%s" type="button" class="btn btn-gold">Contribute</a></p>';
+
+	// Do we have a contribute?
+	if ( $contribute_url && '' !== $contribute_url ) {
+		echo wp_kses( sprintf( $contribute_template, $contribute_url ), wp_kses_allowed_html( 'post' ) );
+	}
+}
+
+/**
+ * Render Footer branding row (logo and social media icons together)
+ *
+ * Determines whether or not we render the logo+social row of the footer.
+ * When this is set to 'disabled' we skip generating the markup altogether,
+ * meaning that individual settings for logo or social icons are ignored.
+ */
+function uds_wp_render_footer_branding_row() {
+	$row_status = get_theme_mod( 'footer_row_branding' );
+
+	if ( 'enabled' === $row_status ) {
+		?>
+		<div class="container" id="endorsed-footer">
+			<div class="row">
+
+				<div class="col-md" id="endorsed-logo">
+					<?php uds_wp_render_footer_logo(); ?>
+				</div>
+
+				<div class="col-md" id="social-media">
+					<div class="social-media-wrapper">
+						<?php
+						if ( has_nav_menu( 'social-media' ) ) {
+							wp_nav_menu(
+								array(
+									'theme_location'  => 'social-media',
+									// 'container' => 'div',
+									// 'container_class' => 'col-md',
+									// 'container_id' => 'social-media',
+									'menu_class'  => '',
+									'items_wrap' => '<nav aria-label="Social Media" class="nav">%3$s</nav>',
+									'walker' => new WP_Social_Media_Walker(),
+								)
+							);
+						}
+						?>
+					</div>
+				</div>
+			</div> <!-- row -->
+		</div> <!-- endorsed-footer -->
+		<?php
+	}
+}
+
+/**
+ * Render 'Contact Us' link in the footer
+ *
+ * Takes the value of the 'contact_url' setting and, if it is not empty,
+ * renders a link with that value as the URL.
+ */
+function uds_wp_render_contact_link() {
+	// =============================
```

Comment:
Looks like an extraneous comment line copy-pasted from the old file. Probably should be dropped.